core/lib/eth_signer/src/raw_ethereum_tx.rs

```diff
@@ -47,6 +48,10 @@ pub struct TransactionParameters {
     pub max_fee_per_gas: U256,
     /// miner bribe
     pub max_priority_fee_per_gas: U256,
+    /// Max fee per blob gas
+    pub max_fee_per_blob_gas: Option<U256>,
+    /// blob versioned hashes
```

Comment:
this comment doesn't say much..

instead - maybe mention about the transaction type. What happens when we use the 'legacy' transactoin type and set this?
Maybe we should change the transction_type to be enum (with blob & blob fee being available only if this is the 4844 transaction?)